Tasks/PublishSymbols/IndexHelpers/IndexFunctions.ps1

```diff
@@ -51,6 +51,13 @@ function Invoke-IndexSources {
                 #    continue
                 #}
 
+                $bytes = Get-Content $symbolsFilePath -Encoding byte -TotalCount 4
```

Comment:
If the file is shorter than four bytes, `$bytes` will have a length of 0, 1, 2, or 3. When decoded as ASCII, it will never match the string "BSJB".